### ISSUE TITLE
make addCacheString optional on marker icon src

### ISSUE THREAD

**Issue by monking** [mapbox/leaflet-image#54]
When loading hundreds or thousands of markers, it often doesn't make sense to add a cache-busting query string. We want to leverage caching in this case.

How about adding a configuration on marker icons (e.g. `allowCache`), and alter [this line](https://github.com/monking/leaflet-image/blob/gh-pages/leaflet-image.js#L116) to something like the following?

```
url = marker.icon.options.allowCache ? marker._icon.src : addCacheString(marker._icon.src),
```

**Comment by tmcw** [comment]
Have you tested this solution? The problem that the cache-busting query attacks is that your browser may have requested a marker image beforehand without CORS headers and has a non-CORS version cached. An optimization will need to continue to solve that problem.

**Comment by monking** [comment]
I have not tested it. How about a single cache-busting value set when leaflet-image loads. I believe this would solve both problems.

**Comment by monking** [comment]
I'm not intimately familiar with the source, so I'll make a pull request after figuring out the solution here.

**Comment by tmcw** [comment]
Fixed in 1b692d6ec4912d27512e2be37ab14d91757c0ce6
